db/migrate/20210930175111_add_alternate_contact.rb

```diff
@@ -0,0 +1,16 @@
+class AddAlternateContact < ActiveRecord::Migration[6.1]
```

Comment:
I'm wondering if we want to change the version of this migration. The string limits migration is at `20211001151212`. So when we are deploying to prod, we need to make sure this migration runs after that one, but the way it is right now, I believe this one will run first, and may push us over the row length limit.

Reply:
Huh I actually did just test this locally starting from `20210902182312` (the migration that adds international telephone), and it seems to work. So looks like the fact that we limit the strings to 200 now in this migration makes it not exceed the limit.

Reply:
Huh I actually did just test this locally starting from `20210902182312` (the migration that adds international telephone), and it seems to work. So looks like the fact that we limit the strings to 200 now in this migration makes it not exceed the limit.

Reply:
Yeah should be fine to leave as is.